Assets/Scripts/Runtime/Interactable/BonusItem.cs

```diff
@@ -0,0 +1,17 @@
+using Assets.Scripts.Runtime;
+using UnityEngine;
+
+public class BonusItem : MonoBehaviour
+{
+    [SerializeField] private BonusItemType _id;
```

Comment:
id oznacza identyfikator czyli coś co określa konkretny obiekt

tutaj to nie jest _id tylko _type

id będzie na UI bo tam to określa jednoznacznie 1 napis na HUDzie